.gitattributes

```diff
@@ -0,0 +1,8 @@
+*.geojson filter=lfs diff=lfs merge=lfs -text
+*.dbf filter=lfs diff=lfs merge=lfs -text
+*.prj filter=lfs diff=lfs merge=lfs -text
+*.sbn filter=lfs diff=lfs merge=lfs -text
+*.sbx filter=lfs diff=lfs merge=lfs -text
+*.shp filter=lfs diff=lfs merge=lfs -text
+*.shp.xml filter=lfs diff=lfs merge=lfs -text
+*.shx filter=lfs diff=lfs merge=lfs -text
```

Comment:
I was weirded out by the `-text` when some of these file types are actually binary, but the [Atlassian Git LFS tutorial](https://www.atlassian.com/git/tutorials/git-lfs) uses binary OGG files as a working example, and those have `-text` as well, so this just seems like a Git LFS thing that we don't have to worry about.